workflows/triage: flag formulae with no ARM bottle

carlocab opened the pull request:
I think it's useful to automatically flag formulae with no ARM bottles as a reminder to investigate whether they can be bottled.

~Nope, didn't work. Must've done this wrong.~

Rylan12 commented:
It looks like the `triage` task is using the workflow on `master` which is why it's not working. See the configuration [here](https://github.com/Homebrew/homebrew-core/pull/68067/checks?check_run_id=1630085345) and note that it doesn't contain the line you added.

It looks right to me, so I'd say revert the formula change and merge and then open a test PR. If that doesn't work, we can always revert/follow-up with a fix.

Also one tiny nitpick: can you move this condition either up one or down one to keep the `deprecate!` and `disable!` check next to each other? 😅

bayandin commented:
Probably, this label shouldn't apply to `bottle :unneeded` formulae.

carlocab commented:
@Rylan12 I had actually meant to do that, I guess my eyes saw double and inserted into the wrong space. 😅 

@bayandin That is a good point. Let me figure out how to do that.

carlocab commented:
Ok, updated. I've set it up so that it'll add the flag for formulae that have a Big Sur bottle but no ARM bottle. At least, I hope that's what I've added does.

Someone let me know if I'm actually mistaken about how this thing works...

dawidd6 commented:
Merge and see if it works as expected. If not, then revert it.
